contracts/shared/ForwarderXReceiver.sol

```diff
@@ -9,7 +9,10 @@ abstract contract ForwarderXReceiver {
     // The Connext contract on this domain
     IConnext public immutable connext;
 
-    /// Modifier
+    /// EVENTS
+    event ForwardedFunctionCallFailed();
```

Comment:
you should at least have the `transferId` in this event so we can filter by transfer ids or some other UUID when consuming

Reply:
Good call